molecule/default/playbook.yml

```diff
@@ -8,7 +8,14 @@
       when: ansible_os_family == 'Debian'
 
   vars:
-    cloud_platform: docker
+    - cloud_platform: docker
+    - ansible_python_interpreter: /usr/bin/python3
 
   roles:
     - role: marvel-nccr.aiida
+        vars:
```

Comment:
```suggestion
      vars:
```

Reply:
And a few other things, extra quotes etc. Fixed now.